Sources/Plasma/Apps/plClient/CMakeLists.txt

```
@@ -246,6 +246,7 @@ target_link_libraries(
         CURL::libcurl
         "$<$<PLATFORM_ID:Darwin>:-framework Cocoa>"
         "$<$<PLATFORM_ID:Darwin>:-framework QuartzCore>"
+        $<$<PLATFORM_ID:Darwin>:LibArchive::LibArchive>
```

```suggestion
        $<$<PLATFORM_ID:Darwin>:${LibArchive_LIBRARIES}>
```

The imported target `LibArhive::LibArchive` wasn't added until CMake 3.17, unfortunately. Our minimum is currently CMake 3.14.